packages/token-lists/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "pancakeswap-token-lists",
+  "version": "2.10.0",
+  "main": "dist/index.js",
+  "type": "module",
+  "license": "MIT",
+  "scripts": {
+    "build": "rm -rf ./dist && rollup -c",
+    "generate:pcs": "yarn test && yarn build && node ./dist > lists/pancakeswap.json",
+    "test": "jest"
+  },
+  "devDependencies": {
+    "@babel/preset-env": "^7.14.1",
+    "@babel/preset-typescript": "^7.13.0",
+    "@ethersproject/address": "^5.1.0",
+    "@rollup/plugin-json": "^4.1.0",
+    "@rollup/plugin-typescript": "^8.2.1",
+    "@types/jest": "^26.0.23",
+    "@types/node": "^15.0.2",
+    "babel-jest": "^26.6.3",
+    "jest": "^26.6.3",
+    "rollup": "^2.47.0",
+    "ts-jest": "^26.5.6"
+  },
+  "dependencies": {
+    "@uniswap/token-lists": "^1.0.0-beta.21",
```

Comment:
do we need this ?

Reply:
This provides validation schema. We could of course copy it here 😅